cmd/enr.go

```diff
@@ -109,3 +113,42 @@ func pubkeyHex(pubkey ecdsa.PublicKey) string {
 func bindEnrFlags(flags *pflag.FlagSet, verbose *bool) {
 	flags.BoolVar(verbose, "verbose", false, "Prints the expanded form of ENR.")
 }
+
+// enrNetworkingKeys returns a string containing the non-empty networking keys (ips and ports) present in the ENR record.
+func enrNetworkingKeys(r enr.Record) string {
```

Comment:
does this work?
```
entries :=  []enr.Entry{
		new(enr.IPv4),
		new(enr.IPv6),
		new(enr.TCP),
		new(enr.TCP6),
		new(enr.UDP),
		new(enr.UDP6),
	}
		
	var sb strings.Builder
	for _, entry := range entries {
		if err := r.Load(entry); err == nil {
			_, _ = sb.WriteString(fmt.Sprintf("%s: %v\n", entry.ENRKey(), entry))
		}
	}
	
	return sb.String()
```

Reply:
unfortunately it doesn't work. your approach seems quite concise though